x-pack/plugins/security_solution/public/management/components/console/components/header_endpoint_info.tsx

```diff
@@ -0,0 +1,103 @@
+import { FormattedDate } from '../../../../common/components/formatted_date';
+import { EndpointHostIsolationStatusProps } from '../../../../common/components/endpoint/host_isolation';
+
+interface HeaderEndpointInfoProps {
+  endpointId: string;
+}
+
+export const HeaderEndpointInfo = memo<HeaderEndpointInfoProps>(({ endpointId }) => {
+  const { data: endpointDetails } = useGetEndpointDetails(endpointId, { refetchInterval: 10000 });
+  const { data: endpointPendingActions } = useGetEndpointPendingActionsSummary([endpointId], {
+    refetchInterval: 10000,
+  });
+
+  const pendingIsolationActions = useMemo<
+    Pick<Required<EndpointHostIsolationStatusProps>, 'pendingIsolate' | 'pendingUnIsolate'>
+  >(() => {
+    if (endpointPendingActions?.data.length) {
+      const pendingActions = endpointPendingActions.data[0].pending_actions;
+
+      return {
+        pendingIsolate: pendingActions.isolate ?? 0,
+        pendingUnIsolate: pendingActions.unisolate ?? 0,
+      };
+    }
+    return {
+      pendingIsolate: 0,
+      pendingUnIsolate: 0,
+    };
+  }, [endpointPendingActions?.data]);
+
+  if (!endpointDetails) {
+    return null;
+  }
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="none">
+      <EuiFlexItem grow={false}>
+        <EuiFlexGroup alignItems="center" gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            <EuiHealth
+              color={HOST_STATUS_TO_HEALTH_COLOR[endpointDetails.host_status]}
+              data-test-subj="consoleHeaderEndpointHealth"
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiText size="s">
+              <h6>
+                <FormattedMessage
+                  id="xpack.securitySolution.console.header.endpointTitle"
+                  defaultMessage="ENDPOINT"
+                />
+              </h6>
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiText color="success" size="s">
+              <h6>{endpointDetails.metadata.host.name}</h6>
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EndpointAgentAndIsolationStatus
+              status={endpointDetails.host_status}
+              isIsolated={endpointDetails.metadata.Endpoint.state?.isolation}
+              {...pendingIsolationActions}
+              data-test-subj="consoleHeaderEndpointIsolationStatus"
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiText color="subdued" size="s">
+          <FormattedMessage
+            id="xpack.securitySolution.console.header.lastSeen"
+            defaultMessage="Last seen "
+          />
+          <FormattedDate
+            fieldName={i18n.translate('xpack.securitySolution.console.header.lastSeen.date', {
+              defaultMessage: 'Last seen',
```

Comment:
Why do we need two `Last seen` sentences? (this and the one above in the `FormattedMessage`)

Reply:
This last seen is for the popover when you hover over the timestamp. There should be enough space to show the entire timestamp, but `fieldName` is a required prop for the `FormattedDate` component